<testsuites>
	<testsuite id="1" tests="2" failures="0" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/auth.spec" time="0.810" timestamp="2025-09-13T16:08:24" name="Auth Actions" errors="0" hostname="Alper-MacBook-Air-2.local">
		<properties></properties>
		<testcase classname="Auth Actions" name="User Register Actions" time="0.642"></testcase>
		<testcase classname="Auth Actions" name="User Login Actions" time="0.168"></testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
	<testsuite id="2" tests="4" failures="4" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/task_crud.spec" time="0.043" timestamp="2025-09-13T16:08:24" name="Task CRUD Actions" errors="0" hostname="Alper-MacBook-Air-2.local">
		<properties></properties>
		<testcase classname="Task CRUD Actions" name="Task Create Actions" time="0.013">
			<failure message="Response status code should be &#34;201&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;201&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;201&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;201&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;201&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="Task CRUD Actions" name="Task Delete Actions" time="0.014">
			<failure message="Response status code should be &#34;204&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;204&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="Task CRUD Actions" name="Task Get Actions" time="0.006">
			<failure message="Response status code should be &#34;200&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;200&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="Task CRUD Actions" name="Task Update Actions" time="0.010">
			<failure message="Response status code should be &#34;202&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;202&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
	<testsuite id="3" tests="5" failures="5" package="/Users/alper/Desktop/taskloom/taskloom-api-test/specs/user_crud.spec" time="0.032" timestamp="2025-09-13T16:08:24" name="User CRUD Actions" errors="0" hostname="Alper-MacBook-Air-2.local">
		<properties></properties>
		<testcase classname="User CRUD Actions" name="User Create Actions" time="0.007">
			<failure message="Response status code should be &#34;201&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;201&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;201&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;201&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;201&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="User CRUD Actions" name="User Delete Actions" time="0.005">
			<failure message="Response status code should be &#34;204&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;204&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;204&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="User CRUD Actions" name="User Get Actions" time="0.006">
			<failure message="Response status code should be &#34;200&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;200&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="User CRUD Actions" name="Get tasks of user" time="0.006">
			<failure message="Response status code should be &#34;200&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;200&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;200&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<testcase classname="User CRUD Actions" name="User Update Actions" time="0.008">
			<failure message="Response status code should be &#34;202&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;401&gt;&#39;" type="Response status code should be &#34;202&#34;&#xA;Concept Execution Failure: &#39;org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;401&gt;&#39;">org.opentest4j.AssertionFailedError: expected: &lt;202&gt; but was: &lt;401&gt;&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#xA;&#x9;at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#xA;&#x9;at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#xA;&#x9;at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:545)&#xA;&#x9;at taskloom.steps.StepImplementation.checkStatusCode(StepImplementation.java:48)&#xA;&#x9;at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#xA;&#x9;at java.base/java.lang.reflect.Method.invoke(Method.java:565)&#xA;&#x9;at com.thoughtworks.gauge.execution.MethodExecutor.execute(MethodExecutor.java:31)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStepMethod(StepExecutionStage.java:66)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.executeStep(StepExecutionStage.java:59)&#xA;&#x9;at com.thoughtworks.gauge.execution.StepExecutionStage.execute(StepExecutionStage.java:41)&#xA;&#x9;at com.thoughtworks.gauge.execution.AbstractExecutionStage.executeNext(AbstractExecutionStage.java:14)&#xA;&#x9;at com.thoughtworks.gauge.execution.HookExecutionStage.execute(HookExecutionStage.java:33)&#xA;&#x9;at com.thoughtworks.gauge.execution.ExecutionPipeline.start(ExecutionPipeline.java:19)&#xA;&#x9;at com.thoughtworks.gauge.processor.ExecuteStepProcessor.process(ExecuteStepProcessor.java:44)&#xA;&#x9;at com.thoughtworks.gauge.RunnerServiceHandler.lambda$executeStep$7(RunnerServiceHandler.java:166)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)&#xA;&#x9;at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)&#xA;&#x9;at java.base/java.lang.Thread.run(Thread.java:1447)&#xA;</failure>
		</testcase>
		<system-out></system-out>
		<system-err></system-err>
	</testsuite>
</testsuites>